packages/@react-aria/utils/src/isElementVisible.ts

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
IMO we should keep this within the focus package for now since it's only used there